cmd/ak/common/consume.go

```diff
@@ -0,0 +1,27 @@
+package common
+
+import (
+	"fmt"
+	"io"
+	"os"
+)
+
+func Consume(args []string) (data []byte, path string, err error) {
```

Comment:
Why not `*string` to avoid the unnecessary default case?

Reply:
not sure i follow, but if i do - you want to check if there is an arg prior to passing it to `Consume`? I think for now this is much more convinient.